gradle-plugin/src/test/groovy/com/google/cloud/tools/dependencies/gradle/LinkageCheckerPluginTest.groovy

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.dependencies.gradle;
+
+import org.junit.Test
+import org.gradle.testfixtures.ProjectBuilder
+import org.gradle.api.Project
+import static org.junit.Assert.*
+
+class LinkageCheckerPluginTest {
+    @Test
+    public void linkage_checker_plugin_should_add_task_to_project() {
+        Project project = ProjectBuilder.builder().build()
+        project.getPlugins().apply 'com.google.cloud.tools.linkageChecker'
+
+        assertTrue(project.tasks.linkageCheck instanceof LinkageCheckTask)
```

Comment:
This test is in Groovy so that it mocks real usage of Gradle project and tasks in build.gradle.